tests/integration/states/azurerm/test_setup_and_teardown.py

```diff
@@ -0,0 +1,171 @@
+import pytest
+import string
+import random
+
+
```

Comment:
is this a catch-all for objects when we require multiples? like additional vnets for testing peering?